homeassistant/components/light/hue.py

```diff
@@ -303,18 +302,6 @@ def turn_on(self, **kwargs):
                 command['sat'] = sat
             else:
                 command['xy'] = kwargs[ATTR_XY_COLOR]
-        elif ATTR_RGB_COLOR in kwargs:
```

Comment:
Removing this is then a breaking change because we no longer set the brightness with `rgb_color`.